azure_monitor/src/azure_monitor/utils.py

```diff
@@ -7,10 +7,16 @@
 import threading
 import time
 
-from opentelemetry.sdk.version import __version__ as opentelemetry_version
+# from opentelemetry.sdk.version import __version__ as opentelemetry_version
+import pkg_resources
 
 from azure_monitor.version import __version__ as ext_version
 
+# Workaround for missing version file
```

Comment:
Is this a temporary workaround?

Reply:
Yes, I believe someone is trying to make the push in OT to remove `version.py` from the packages. If this happens, we will use this workaround, but if not, we will wait until the `version.py` file is put back in the release and use that instead.